english/friendly-person/panic/Scrabble/Scrabble.md

```diff
@@ -0,0 +1,3 @@
+You scrabble desperately to get away, but as you do you see that it was actually the box for the board game "Scrabble"...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.